src/main/java/com/minecolonies/api/util/constant/SchematicTagConstants.java

```diff
@@ -5,23 +5,77 @@
  */
 public class SchematicTagConstants
 {
-    public static final String TAG_SITTING = "sit";
-    public static final String TAG_WORK    = "work";
+    /**
+     * General tags.
+     */
+    public static final String TAG_SITTING   = "sit";
+    public static final String TAG_WORK      = "work";
+    public static final String TAG_SIT_IN    = "sit_in";
+    public static final String TAG_SIT_OUT   = "sit_out";
+    public static final String TAG_STAND_IN  = "stand_in";
+    public static final String TAG_STAND_OUT = "stand_out";
+    public static final String BUILDING_SIGN = "building_sign";
```

Comment:
We could move miner level sign to tag too honestly, would be breaking change though, so keep both codes at least two majors